### PR TITLE
Pushing my data to the GDSC-Lead Map

### PR DESCRIPTION
**Pull request by JuliusKanneh**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ import karan from "../../photos/Karan.jpg";
 import ishubham21 from "../../photos/ishubham21.jpg"
 import rohitdalvi18 from "../../photos/rohitdalvi18.jpg";
 import jestinroy3 from "../../photos/jestinroy3.jpg";
+import abhinavsri360 from "../../photos/JuliusKanneh.jpg";
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:76]:
```suggestion
import JuliusKanneh from "../../photos/JuliusKanneh.jpg";
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@JuliusKanneh import isn't correct

File: src/components/json/worldData.js [hunk 2]
```diff
@@ -1281,6 +1282,22 @@ const data = [
 		twitter: "https://twitter.com/thejestinroy",
 		portfolio: "https://jestinroy3.netlify.app/"
 	},
+{
+		longitude: -1.981760,
+		latitude: 30.035740,
+		batch: 2022,
+		photo: JuliusKanneh,
+		region: "Africa",
+		skills: ["Web Dev", "IOT"],
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:1291]:
```suggestion
		skills: ["Web Dev", "IoT"],
```

**Comment by abhinavsri360** [comment]
Thanks, @JuliusKanneh for submitting your profile. Thanks, @aravindvnair99 for reviewing.
Sorry for the late response.

**Pull request merged**